### PR TITLE
feature: Using bentoml1.0 apis

### PR DESCRIPTION
**Pull request by jjmachan**

### CODE REVIEW

File: bentoctl_aws_ec2/utils/__init__.py [hunk 1]
```diff
@@ -91,3 +96,14 @@ def push_docker_image_to_repository(
         docker_client.images.push(**docker_push_kwags)
     except docker.errors.APIError as error:
         raise Exception(f"Failed to push docker image {image_tag}: {error}")
+
+
+def get_tag_from_path(path):
```
Comment (parano) [inline bentoctl_aws_ec2/utils/__init__.py:101]: Use 
```
bento = Bento.from_fs( fs.open_fs(path) )
return bento.tag
```

Comment (jjmachan) [reply bentoctl_aws_ec2/utils/__init__.py:101]: we have to load `Bento` from bentoml._internal.bento right, or will it be exposed somewhere else?

File: bentoctl_aws_ec2/deploy.py [hunk 2]
```diff
@@ -31,7 +30,7 @@ def deploy(bento_bundle_path, deployment_name, ec2_config):
 
     # make deployable folder and overide if found according to user pref
     project_path = os.path.join(
-        os.path.curdir, f"{bento_metadata.name}-{bento_metadata.version}-deployable"
+        os.path.curdir, f"{bento_tag.name}-{bento_tag.version}-deployable"
```
Comment (yubozhao) [inline bentoctl_aws_ec2/deploy.py:33]: Make sure this is using the deployment name rather than bento info

File: bentoctl_aws_ec2/deploy.py [hunk 3]
```diff
@@ -48,13 +47,15 @@ def deploy(bento_bundle_path, deployment_name, ec2_config):
...
-            ec2_config["region"], repo_name
+            deployment_spec["region"], repo_name
+        )
+        _, username, password = get_ecr_login_info(
+            deployment_spec["region"], repository_id
         )
-        _, username, password = get_ecr_login_info(ec2_config["region"], repository_id)
         ecr_tag = generate_docker_image_tag(
-            registry_url, bento_metadata.name, bento_metadata.version
+            registry_url, bento_tag.name, bento_tag.version
```
Comment (yubozhao) [inline bentoctl_aws_ec2/deploy.py:56]: let's get this info from asking user, rather than construct it from the bento info

Comment (yubozhao) [reply bentoctl_aws_ec2/deploy.py:56]: We can have the default to bento tag

Comment (yubozhao) [reply bentoctl_aws_ec2/deploy.py:56]: Let's add the docker_image_tag as part of the deployment spec as an optional field, with the default to bento tag

Comment (yubozhao) [reply bentoctl_aws_ec2/deploy.py:56]: the default will be {bento-name}:{bento-version}

File: bentoctl_aws_ec2/utils/__init__.py (hunk 1 above)
Comment (yubozhao) [reply bentoctl_aws_ec2/utils/__init__.py]: from bentoml.bentos import Bento

**Pull request merged**